### PR TITLE
Add support for redirecting to the web RP when the web session expires

### PR DESCRIPTION
**Pull request by gene1wood**
This has a default of the Mozilla federated RP. We should probably establish a more centralized method to enumerate defaults. When we do we can move this default there.

This also doesn't do anything for refreshing the CLI session.

### CODE REVIEW

File: federated_aws_cli/login.py [hunk 1]
```diff
@@ -337,21 +340,29 @@ def aws_federate(self):
...
+        role = self.role_arn.split(':')[5].split('/')[-1]
+        issuer_url_query = urlencode({"account": account_id, "role": role})
+        issuer_url = urlunparse(
+            ('https', self.issuer_domain, '/', '', issuer_url_query, ''))
+
+        query = urlencode({
+            "Action": "login",
+            "Destination": "https://console.aws.amazon.com/",
+            "SigninToken": token["SigninToken"],
+            "Issuer": issuer_url
```
Comment (april) [inline federated_aws_cli/login.py]: Trailing comma.  :)

File: federated_aws_cli/login.py [hunk 2]
```diff
@@ -337,21 +340,29 @@ def aws_federate(self):
...
+        })
+        url = urlunparse(url_tuple._replace(query=query))
```
Comment (april) [inline federated_aws_cli/login.py]: What is `_.replace()`?

File: federated_aws_cli/login.py [hunk 3]
```diff
@@ -54,6 +55,7 @@ def __init__(
         scope="openid",
         token_endpoint="https://auth.mozilla.auth0.com/oauth/token",
         web_console=False,
+        issuer_domain=None
```
Comment (april) [inline federated_aws_cli/login.py]: Trailing comma.  :)

File: federated_aws_cli/login.py (hunk 2 above)
Comment (gene1wood) [reply federated_aws_cli/login.py]: [`_replace`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse) is a method on a parsed URL which the docs recommend using to change values. Why it's a private method is beyond me
> As is the case with all named tuples, the subclass has a few additional methods and attributes that are particularly useful. One such method is _replace(). The _replace() method will return a new ParseResult object replacing specified fields with new values.

File: federated_aws_cli/login.py (hunk 1 above)
Comment (gene1wood) [reply federated_aws_cli/login.py]: Fixed in d12afa1854849ffbb40cf1c2a7e77458ccf7ce54

File: federated_aws_cli/login.py (hunk 3 above)
Comment (gene1wood) [reply federated_aws_cli/login.py]: Fixed in d12afa1854849ffbb40cf1c2a7e77458ccf7ce54

**Pull request merged**